### DISCUSSION TITLE
Sometimes my next-server returns an empty response and logs aren't helpful

### DISCUSSION THREAD

**Discussion by insanity54** [vercel/next.js#56838] [Help]
### Summary

I'm running a next project with next 13.5.4. I'm running a standalone next-server upstream from a CDN in production and it runs as expected for many hours. Intermittently, the server will return an empty `text/html; charset=utf-8` response with status code 304 on only the root route. 

The 304 is persistent. If I don't catch this within a few hours, and my CDN cache refreshes, this causes a partial outage on my production site, where visitors see a blank white page where content is expected.

Restarting my next server temporarily fixes the problem. The front page again gets a full html response, with status code 200.

I have not been able to troubleshoot this problem, since the next-server logs are redacted.

```
2023-10-13T09:32:11.875389498Z app[web.1]: [Error: An error occurred in the Server Components render. The specific message 
is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.] {
2023-10-13T09:32:11.875400750Z app[web.1]:   digest: '956933652'
2023-10-13T09:32:11.875403374Z app[web.1]: }
```

I don't appreciate this error message. I am expecting to look into server logs and find the cause of the error, but instead, the error is hidden from me. All I can learn from the logs is that there was an error, but it doesnt' tell me why there was an error, and this is frustrating. This log is useless to me. 

I have not been able to replicate the issue in development. I only see this empty response behavior in production, after the server is running for hours, sometimes days. Four times a week I've seen this issue happen and every time I dig into my component code I have no idea what to look for.

I need more information. I need the contents of the server log. Is there a way to configure the server to show the full content of it's production logs?

**Comment by alvvaysxxx** [comment]
same issue, work perfectly on my local machine, but vercel messed it up as expected
